src/clj/game/cards/programs.clj

```diff
@@ -1537,17 +1537,10 @@
 
 (defcard "Leech"
   {:events [{:event :successful-run
-             :interactive (get-autoresolve :autofire (complement never?))
-             :silent (get-autoresolve :autofire never?)
-             :optional
-             {:player :runner
-              :req (req (is-central? (target-server context)))
-              :waiting-prompt "Runner to decide if they will use Leech"
-              :autoresolve (get-autoresolve :auto-fire)
-              :prompt "Use Leech?"
-              :yes-ability {:msg "add 1 virus counter to Leech"
-                            :effect (req (add-counter state side card :virus 1))}
-              :no-ability {:effect (req (system-msg state side "does not add counter to Leech"))}}}]
+             :interactive (get-autoresolve :auto-fire (complement never?))
```

Comment:
Oops, I led you astray! You can remove this and the `:silent` entirely. They're only needed with `:optional` stuff.

Reply:
I removed both and added `:req` which I believe is necessary? Going to take a go at tests next!